### PR TITLE
Improve displaying of rating given license state or errors. display error in price ticket.

### PR DESCRIPTION
**Pull request by iurimatias**

### CODE REVIEW

File: app/js/components/License.jsx [hunk 1]
```diff
@@ -26,7 +26,9 @@ const License = (props) => (
     <CardBody>
       {props.error && <Alert color="danger">{props.error}</Alert>}
       {props.isLicenseOwner ? <IsLicenseOwner t={props.t}/> : <BuyLicense buyLicense={props.buyLicense} t={props.t}/>}
-      <p>{props.t('license.rating')} {props.userRating ? props.userRating : '-'}</p>
+      {props.isLicenseOwner &&
+        <p>{props.t('license.rating')} {props.userRating ? props.userRating : 'no rating available yet'}</p>
```
Comment (jrainville) [inline app/js/components/License.jsx]:
```suggestion
        <p>{props.t('license.rating')} {props.userRating ? props.userRating : t('license.noRating')}</p>
```
Also, please capitalize the sentence

File: app/js/containers/PriceContainer.jsx [hunk 2]
```diff
@@ -1,20 +1,28 @@
...
 
 import Price from '../components/Price';
 
-const PriceContainer = ({ ethUsd, sntUsd }) => (
+const PriceContainer = ({ ethUsd, sntUsd, hasErrors }) => (
   <div>
-    <Price logo="https://raw.githubusercontent.com/TrustWallet/tokens/master/coins/60.png" priceTicker="ETH/USD" price={ethUsd} />
-    <Price logo="https://raw.githubusercontent.com/TrustWallet/tokens/master/images/0x744d70fdbe2ba4cf95131626614a1763df805b9e.png" priceTicker="SNT/USD" price={sntUsd} />
+    {hasErrors &&
+      <p>Could not fetch prices</p>
```
Comment (jrainville) [inline app/js/containers/PriceContainer.jsx]: Could you add the translation module please.

File: app/js/features/prices/reducer.js [hunk 3]
```diff
@@ -7,12 +7,18 @@ function reducer(state = {}, action) {
...
   }
 }
 
 export default reducer;
 
-export const getEthUsdPrice = state => state.prices.ETH.USD;
-export const getSntUsdPrice = state => state.prices.SNT.USD;
+export const getEthUsdPrice = state => state.prices.ETH && state.prices.ETH.USD;
+export const getSntUsdPrice = state => state.prices.SNT && state.prices.SNT.USD;
+export const hasPricesError = state => !!state.prices.error
```
Comment (jrainville) [inline app/js/features/prices/reducer.js]: I would change this to `getPricesError` and in the component, you can still check for it's truthiness, but also display the actual error.

File: stories/license.stories.jsx [hunk 4]
```diff
@@ -1,20 +1,33 @@
...
+import { withKnobs, text, number } from '@storybook/addon-knobs';
 import { withInfo } from "@storybook/addon-info";
 import { action } from '@storybook/addon-actions';
 
 import License from '../app/js/components/License';
 
-storiesOf('License', module)
+const stories = storiesOf('License', module)
+
+stories.addDecorator(withKnobs);
```
Comment (jrainville) [inline stories/license.stories.jsx]: What do knobs do?

Comment (alaibe) [reply stories/license.stories.jsx]: It allows to make change to the props directly in storybook

**Pull request merged**